molo/core/models.py

```diff
@@ -511,6 +511,22 @@ class SectionPage(CommentedPageMixin, TranslatablePageMixin, Page):
         null=True, blank=True,
         help_text='The date rotation will end')
 
+    enable_next_section = (
+        models.BooleanField(
+            default=False,
+            verbose_name='Activate up next section underneath articles',
+            help_text=("Activate up next section underneath articles in this "
+                       "section will apear with the heading and subheading of"
+                       " that article. The text will say 'next' in order to "
+                       "make the user feel like it's fresh content.")))
```

Comment:
Not sure if we need the explanation of what the text will say and why. Things like these are great to say to everyone when we are doing the demo